MVP/defaults-build.yaml

```diff
@@ -26,6 +26,8 @@
         - timestamps
         - build-name:
             name: ${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}
+        - inject:
+            script-content: echo "create_time=$(date --utc +%FT%TZ)" >> $WORKSPACE/job.properties
```

Comment:
I just applied it to some jobs and re-run one of them.
Hope it's caught as expected.